src/kvstore/raftex/Host.cpp

```diff
@@ -303,7 +304,8 @@ void Host::appendLogsInternal(folly::EventBase* eb,
                         r.set_error_code(cpp2::ErrorCode::SUCCEEDED);
                         self->setResponse(r);
                     } else {
-                        self->lastLogIdSent_ = resp.get_last_log_id();
+                        self->lastLogIdSent_ = std::min(resp.get_last_log_id(),
```

Comment:
If SUCCEEDED returned,  it means follower accept all logs,  so resp.get_last_log_id() should be less than self->logIdToSend_, right?

Reply:
Yeah, so when SUCCEEDED returned, we just set `lastLogIdSent_ = resp.get_last_log_id()`, but in E_LOG_GAP, I think we'd better compare them?

Reply:
Fine.  Look the same.